app/src/main/res/values-fr/strings.xml

```diff
@@ -82,4 +84,17 @@
     <string name="auth_summary_title">Vous êtes connecté!</string>
     <string name="auth_summary_to">vers:</string>
 
+    <!-- Identity -->
+    <string name="identity_title">Identités</string>
+    <string name="identity_blocked">Bloqué</string>
+    <string name="identity_blocked_message">Attention: ce compte est bloqué et doit être re-activé.</string>
+    <string name="identity_delete_title">Êtes-vous sûr?</string>
+    <string name="identity_delete_message">Ce compte ne peut plus être utilisé lorsqu\'il est supprimé.</string>
+    <string name="identity_biometric_usage">Utiliser la biométrie</string>
+    <string name="identity_biometric_upgrade">Mise à niveau vers l\'utilisation de la biométrie</string>
+
```

Comment:
Lots of empty lines

Reply:
There will be more coming soon, filling up the empty lines :blush: